### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by L-tech**
https://github.com/spheronFdn/fns/issues/34

### CODE REVIEW

**Review by aayushmahapatra — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -128,8 +142,19 @@ For addresses, this means the "match" property (a boolean value for matching rev
...
-/* Get Blockchain Address
-Returns the Blockchain address associated with the provided node and coinType, or 0 if none. */
+ /* Get Blockchain Address
+
+Get Blockchain Address
+🔍 Need to find the blockchain address associated with your FNS name and coinType? 🔍
+
+Our handy function can help! Simply provide the node (FNS name) and coinType, and we'll return the associated blockchain address. If no address is found, don't worry - the function will return 0. 😎
+
+At the moment, FNSlib supports BTC, LTC, DOGE, and FIL, but we're always looking to add support for more major chains in the future. If you have a specific request for a coinType, head on over to our 💬 [discord channel](https://discord.com/invite/ahxuCtm) and let us know. We're always considering new ideas for subsequent updates. 🚀
```
Comment (rekpero) [inline README.md:152]:
```suggestion
At the moment, FNSlib supports BTC, LTC, DOGE, and native FIL, but we're always looking to add support for more major chains in the future. 

If you have a specific request for a coinType, head to our 💬 [discord channel](https://discord.com/invite/ahxuCtm) and let us know. We're always considering new ideas for subsequent updates. 🚀
```

File: README.md [hunk 2]
```diff
@@ -163,8 +188,21 @@ const  address  =  await FNSInstance.getAddress('juan.fil', BlockchainIdentifier
 const  address  =  await FNSInstance.getAddress('juan.fil', BlockchainIdentifier.DOGECOIN)
 
 
-/* Normal content fetching */
-const contentHash = await FNSInstance.getContent('juan.fil')
+/* Normal content fetching 
+To get content hash attached to a fns name
+*/
+
+const  contentHash  =  await FNSInstance.getContent('juan.fil')
```
Comment (rekpero) [inline README.md:195]: Apart from this, show how to get the name from address using FNSLib and describe the flow of how to use it

**Review by rekpero — CHANGES_REQUESTED** [review]
some changes needed in the doc

**Review by rekpero — APPROVED** [review]

**Review by aayushmahapatra — APPROVED** [review]

**Pull request merged**